src/App/Controls/CipherViewCell/CipherViewCellViewModel.cs

```diff
@@ -6,11 +6,29 @@ namespace Bit.App.Controls
     public class CipherViewCellViewModel : ExtendedViewModel
     {
         private CipherView _cipher;
+        private bool _websiteIconsEnabled;
+
+        public CipherViewCellViewModel(CipherView cipherView, bool websiteIconsEnabled)
+        {
+            Cipher = cipherView;
+            WebsiteIconsEnabled = websiteIconsEnabled;
+        }
 
         public CipherView Cipher
         {
             get => _cipher;
             set => SetProperty(ref _cipher, value);
         }
+
+        public bool WebsiteIconsEnabled
+        {
+            get => _websiteIconsEnabled;
+            set => SetProperty(ref _websiteIconsEnabled, value);
+        }
+
+        public bool ShowIconImage
+        {
+            get => WebsiteIconsEnabled && Cipher.Login != null && !string.IsNullOrWhiteSpace(Cipher.Login.Uri);
```

Comment:
This _could_ be simplified:
```suggestion
            get => WebsiteIconsEnabled && !string.IsNullOrWhiteSpace(Cipher.Login?.Uri);
```
```suggestion
            get => WebsiteIconsEnabled && Cipher.Login != null && !string.IsNullOrWhiteSpace(Cipher.Login.Uri);
```

Reply:
Simplified plus added 'http' check (bug discovered while working on #1384 )